app/assets/stylesheets/test.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+    font-size: 200px;
```

Comment:
Check the indentation.

Reply:
Yes, I'll check this to use an indentation of 2 spaces